### PR TITLE
Using payable for buyTicket contract method

### PR DESCRIPTION
**Pull request by tetratorus**

### CODE REVIEW

File: server/ethereum/ethserver.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+app.get('/', (req, res) => {
+  res.send('hello world');
+});
+
+app.post('/api/createEvent', (req, res) => {
+  ethController.createEvent(req, res);
+});
+
+app.post('/api/buyTicket', (req, res) => {
+  console.log('BODY:', req);
```
Comment (andrewk17) [inline server/ethereum/ethserver.js]: remove console log

File: server/ethereum/services/buySvc.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+let Web3 = require('web3');
```
Comment (andrewk17) [inline server/ethereum/services/buySvc.js]: these let's can probably be const's

File: server/ethereum/services/buySvc.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+let solc = require('solc');
+let web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
```
Comment (andrewk17) [inline server/ethereum/services/buySvc.js]: require the web3js file instead so we have one point of instantiation

File: server/ethereum/services/buySvc.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+let fs = require('fs');
+
+let buySvc = {
+  buyTicket: (req, res) => {
+    console.log('ADDRESS IS', req.body.contractAddress);
```
Comment (andrewk17) [inline server/ethereum/services/buySvc.js:10]: remove console log

File: server/ethereum/services/createSvc.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ const solc = require('solc');
 const web3Connection = require('../web3.js');
 const web3 = web3Connection.web3;
 
-const senderAddress = web3.eth.accounts[0];
+var senderAddress = web3.eth.accounts[0];
```
Comment (andrewk17) [inline server/ethereum/services/createSvc.js]: any reason this was changed to var?

File: server/server.js [hunk 6]
```diff
@@ -21,26 +20,23 @@ app.get('/host', function(req, res) {
...
 })
 
-app.post('/api/createEvent', (req, res) => {
-  ethController.createEvent(req, res);
-});
+app.listen(config.SERVER_PORT);
 
-const server = app.listen(port);
-console.log('Listening at port: ' + port);
+var Web3 = require('web3');
```
Comment (andrewk17) [inline server/server.js:25]: do we need these web3 references?

**Review by andrewk17 — CHANGES_REQUESTED** [review]

File: Gruntfile.js [hunk 7]
```diff
@@ -16,10 +16,13 @@ module.exports = function(grunt) {
         stderr: false
       },
       compile: {
-        command: 'node ./node_modules/webpack/bin/webpack.js'
+        command: 'node ./node_modules/webpack/bin/webpack.js '
```
Comment (chrispicato) [inline Gruntfile.js]: why is there a space?

**Review by chrispicato — CHANGES_REQUESTED** [review]

File: server/ethereum/ethserver.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+app.post('/api/buyTicket', (req, res) => {
+  ethController.buyTicket(req, res);
+})
+
+const server = app.listen(config.ETH_SERVER_PORT, function() {
+  console.log('Running on', config.ETH_SERVER_PORT);
+});
+
+let Web3 = require('web3');
```
Comment (andrewk17) [inline server/ethereum/ethserver.js]: Do we need these references to Web3?

**Review by chrispicato — APPROVED** [review]

**Review by kevinbrosamle — APPROVED** [review]

**Review by andrewk17 — APPROVED** [review]

**Pull request merged**